tests/ci/pr_info.py

```diff
@@ -215,12 +217,12 @@ def __init__(
                     # files changed in upstream AND master...{self.head_ref}
                     # to get files, changed in current HEAD
                     self.diff_urls.append(
-                        f"https://github.com/{GITHUB_REPOSITORY}/"
-                        f"compare/master...{self.head_ref}.diff"
+                        f"https://api.github.com/repos/{GITHUB_REPOSITORY}/"
+                        f"compare/master...{self.head_ref}"
                     )
                     self.diff_urls.append(
-                        f"https://github.com/{GITHUB_REPOSITORY}/"
-                        f"compare/{self.head_ref}...master.diff"
+                        f"https://api.github.com/repos/{GITHUB_REPOSITORY}/"
+                        f"compare/{self.head_ref}...master"
```

Comment:
And here and below we need to use `pull_request["head"]["label"]`

Reply:
Yes. I wonder why self.head_ref worked before here.